x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccounts.java

```diff
@@ -28,8 +28,13 @@
             new RoleDescriptor.IndicesPrivileges[]{
                 RoleDescriptor.IndicesPrivileges
                     .builder()
-                    .indices("logs-*", "metrics-*", "traces-*")
+                    .indices("logs-*", "metrics-*", "traces-*", ".logs-endpoint.diagnostic.collection-*")
                     .privileges("write", "create_index", "auto_configure")
+                    .build(),
+                RoleDescriptor.IndicesPrivileges
+                    .builder()
+                    .indices(".fleet-*")
+                    .privileges("read", "write", "monitor", "create_index", "auto_configure")
```

Comment:
The PR description says `all`, but I much prefer explicitly listing the privileges that are needed.
Can we update the description?

Reply:
Indeed the description is outdated. It's updated now. Thanks!